ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -736,24 +693,26 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
         }
     }
 
-    /// Override to set a specific `CustomViewSize` on the `customView`
-    @objc open var customViewSize: CustomViewSize {
+    /// Override to set a specific `MSFTableViewCellCustomViewSize` on the `customView`
+    @objc open var customViewSize: MSFTableViewCellCustomViewSize {
         get {
             if customView == nil {
                 return .zero
             }
-            return _customViewSize == .default ? layoutType.customViewSize : _customViewSize
+            tokens.customViewSize = _customViewSize == .default ? layoutType.customViewSize : _customViewSize
```

Comment:
Won't this mean that an overriden token set with a custom value for `customViewSize` will never be respected?

Reply:
Ah I understand, that's a state prop rather than a measurement. I was expecting something of type `CGSize`. I wonder if maybe this value should be named something else?

Reply:
Actually I don't think adding `Type` is great here because that word is never used anywhere else. I think we should probably stick with the original naming for now; consistency is more important here.